metadata/example/SDasDataset.json

```
@@ -1,17 +1,35 @@
 {
     "schema": "https://github.com/biocaddie/DATS/blob/master/dataset_schema.json",
-    "title": "SDas Phantom",
-    "description": "Samir Das phantom scans",
+    "title": "Humain MRI Phantom",
```

`Human` instead of `Humain`. A better title would be `Multicenter Human MRI Phantom`, the multicenter part is what makes this dataset attractive.

---

Agreed, something  like

```suggestion
    "title": "Human MRI Phantom",
    "description": "Samir Das phantom scans, T1 weighted images on many different sites",
```

Also,  if we do searches on the "description" field it would be good to put relevant keywords there - for instance I think this could be considered as a "longitudinal" study and if we know the other type of images we could add them in the description as well